starter-code/public/javascript/APIHandler.js

```diff
@@ -3,23 +3,110 @@ class APIHandler {
     this.BASE_URL = baseUrl;
   }
 
-  getFullList () {
+  // See full list
 
-  }
+  getFullList () { 
+    axios.get(`${this.BASE_URL}/characters`)
+    .then((response) => {
+      let mainContainer = document.getElementsByClassName('characters-container');
+      mainContainer[0].innerHTML = '';
 
-  getOneRegister () {
+      response.data.forEach(element => {
+        let text = `
+        <div class="character-info">
+        <div class="name">Name: ${element.name}</div>
+        <div class="occupation">Occupation: ${element.occupation}</div>
+        <div class="cartoon">Is a Cartoon?</div>
+        <div class="weapon">Weapon: ${element.weapon}</div>
+        </div>`;
+        mainContainer[0].innerHTML += text;
+      })
+    })
+    .catch(err => {
+      console.log(err);
+    })
+};
 
-  }
+// See one character
+  
+getOneRegister() {
+  const id = $('input[name=character-id]').val();
 
-  createOneRegister () {
+  axios.get(`${this.BASE_URL}/characters/${id}`)
+    .then((response) => {
+      let mainContainer = document.getElementsByClassName('characters-container');
+      mainContainer[0].innerHTML = '';
 
-  }
+      let text = `
+      <div class="character-info">
+      <div class="name">Name: ${response.data.name}</div>
+      <div class="occupation">Occupation: ${response.data.occupation}</div>
+      <div class="cartoon">Is a Cartoon? ${response.data.cartoon}</div>
+      <div class="weapon">Weapon: ${response.data.weapon}</div>
+      </div>`;
 
-  updateOneRegister () {
+      containerPrincipal[0].innerHTML = text;
+    })
 
-  }
+    .catch(err => {
+      console.log(err);
+    })
+}
 
-  deleteOneRegister () {
+// Create new carachter
 
-  }
+createOneRegister() {
+  const name = $('input[name=name-create]').val();
```

Comment:
Would be cleaner to extract all information from the DOM in the index.js. In this way your API handler file would be only handling API calls.